src/crimson/os/seastore/omap_manager.h

```diff
@@ -14,8 +14,8 @@
 #include "crimson/os/seastore/seastore_types.h"
 #include "crimson/os/seastore/transaction_manager.h"
 
-#define OMAP_INNER_BLOCK_SIZE 4096
-#define OMAP_LEAF_BLOCK_SIZE 8192
+#define OMAP_INNER_BLOCK_SIZE 8192
+#define OMAP_LEAF_BLOCK_SIZE 65536
```

Comment:
Worth a TODO here to calculate the max key and value sizes the current layout supports, and return errors during insert if the max is exceeded, rather than breaking the code somewhere else.

Reply:
Yeah, will do.

Reply:
Done.